_config.yml

```diff
@@ -15,14 +15,15 @@
 # in the templates via {{ site.myvariable }}.
 title: Open Targets
 author: Open Targets Team
-email: support@targetvalidation.org
+email: contact@opentargets.org
 description: > # this means to ignore newlines until "baseurl:"
   Open Targets is a public-private initiative to generate evidence on the validity of therapeutic targets based on genome-scale experiments and analysis. 
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "" # the base hostname & protocol for your site, e.g. http://example.com
 twitter_username: targetvalidate
 github_username:  opentargets
-
+linkedin: https://www.linkedin.com/company/centre-for-therapeutic-target-validation
```

Comment:
i moved all the repeated values inside the config.yml so that it's easier to keep consistent across pages in the future.